infer/src/pulse/PulseBaseDomain.ml

```diff
@@ -125,7 +125,7 @@ module GraphComparison = struct
             IsomorphicUpTo mapping
         | Some _, None | None, Some _ ->
             NotIsomorphic
-        | Some (edges_rhs, attrs_rhs), Some (edges_lhs, attrs_lhs) ->
+        | Some (edges_lhs, attrs_lhs), Some (edges_rhs, attrs_rhs) ->
```

Comment:
This one was clearly a bug, great catch! I'm curious if you have a test case where it caused issues, we could add it to our tests if so but otherwise it's ok too.